TombEngine/Sound/sound.cpp

```diff
@@ -628,6 +628,15 @@ int Sound_EffectIsPlaying(int effectID, Pose *position)
 	return -1;
 }
 
+bool Sound_EffectIsPlaying(int effectID)
+{
+	if (effectID == -1)
+		return;
```

Comment:
This doesn't really serve any purpose. The function should return `false` if `channelIndex` is -1. ;)

Reply:
done :)